schema/readout/fakecardreader.jsonnet

```diff
@@ -55,7 +58,9 @@ local fakecardreader = {
         s.field("queue_name", self.string,
             doc="Name of the output queue"),
         s.field("random_population_size", self.uint4, 10000,
-            doc="Size of the random population")
+            doc="Size of the random population"),
+        s.field("frame_error_rate", self.double8, 0.0,
+            doc="Rate of errors in WIB frame header"),
```

Comment:
Not only WIB. With extending the interface of readout types for faking errors, it should work also DAPHNE and any other FE types.